### PR TITLE
Clean up 3d-tiles-tools project and dependencies

### PR DESCRIPTION
**Pull request by mramato**
1. Update all package.json dependencies and switch to caret versioning.
2. Switched to `klaw`, which used to be bundled with `fs-extra` but they removed it and directed users to use it directly.
2. Clean up Webstorm Project, which was severely out of date.
3. Fix coverage.

### CODE REVIEW

**Comment by mramato** [comment]
Travis is going to fail because it was enabled without having a `.travis.xml` set up in the root folder.  This will be fixed in a separate PR (probably by someone else).

File: tools/gulpfile.js [hunk 1]
```diff
@@ -72,7 +72,11 @@ gulp.task('coverage', function () {
         ' cover' +
         ' --include-all-sources' +
         ' --dir coverage' +
-        ' -x "bin/** -x doc/** -x specs/** -x coverage/** -x index.js -x gulpfile.js"' +
+        ' -x "doc/**"' +
+        ' -x "specs/**"' +
+        ' -x "coverage/**"' +
+        ' -x "gulpfile.js"' +
+        ' -x "index.js"' +
```
Comment (shehzan10) [inline tools/gulpfile.js]: Won't skipping the `bin` directory lower the coverage stats?

**Comment by shehzan10** [comment]
> Travis is going to fail because it was enabled without having a .travis.xml set up in the root folder. This will be fixed in a separate PR (probably by someone else).

I think we can pretty much copy the entire [gltf-pipeline/.travis.yml](https://github.com/AnalyticalGraphicsInc/gltf-pipeline/blob/master/.travis.yml).

File: tools/gulpfile.js (hunk 1 above)
Comment (mramato) [inline tools/gulpfile.js]: We aren't skipping the `bin` directory.  And bin files can be programmatically unit tested just like the rest of the library so there's no reason to exclude them.

**Comment by mramato** [comment]
> I think we can pretty much copy the entire gltf-pipeline/.travis.yml.

It's not that simple.  Since we have 3 separate npm packages in this repository, we need a top-level travis.yml that knows about each one and makes it easy to maintain and separate what each one does.  It's not complicated, but definitely worth a separate PR to implement.

**Comment by mramato** [comment]
> Moved klaw to dependencies

Thanks, totally missed that.

**Comment by lilleyse** [comment]
Thanks, looks good. I'll open an issue for fixing travis.

**Comment by lilleyse** [comment]
Published to npm.

**Comment by mramato** [comment]
Awesome, thanks.

**Pull request merged**